runtests.sh

```diff
@@ -280,6 +283,14 @@ do
             doRuffFormat=true
             doCopyRight=true
         ;;
+        --formatfix)
+            doIsortFix=true
+            doBlackFix=true
+            doRuffFix=true
+            doIsortFormat=true
+            doBlackFormat=true
+            doRuffFormat=true
```

Comment:
the `print_style_fail_msg` doesn't give a correct instruction for non default directory, I think we can ignore

Reply:
Yes, updated in this [commit](https://github.com/Project-MONAI/MONAI/pull/7106/commits/571b62bfa8c5c494b52369557bed5a0043cedc72). Thanks!